cmd/osbuild-tests/main_test.go

```diff
@@ -0,0 +1,343 @@
+// +build integration
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/BurntSushi/toml"
+	"github.com/google/uuid"
+	"github.com/osbuild/osbuild-composer/internal/blueprint"
+
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestComposeIntegration(t *testing.T) {
+	// Smoke tests, until functionality tested fully
+	// that the calls succeed and return valid output
+	runComposerCLI(t, false, "compose", "types")
+	runComposerCLI(t, false, "compose", "status")
+	runComposerCLI(t, false, "compose", "list")
+	runComposerCLI(t, false, "compose", "list", "waiting")
+	runComposerCLI(t, false, "compose", "list", "running")
+	runComposerCLI(t, false, "compose", "list", "finished")
+	runComposerCLI(t, false, "compose", "list", "failed")
+	// runCommand(false, "compose", "log", UUID, [<SIZE>])
```

Comment:
Ideally these will be split into several testing functions, one for each command. In case there are more scenarios for sub-commands then we can split again.  However this can be done as a follow up.

Reply:
Could you please elaborate here? I'm not sure I've understood the suggestion correctly - if that means replacing `runComposerCLI(t, x, y, z)` with new specialized functions, it would seem more difficult to maintain and understand. Is it so or you meant something different? :)

Reply:
It means not calling `runComposerCLI` for all possible commands/sub-commands inside a single function.  Some of the commented out lines will need their own setup and makes sense for them to be separated. 

However this is not related to the current changes and can be done as a follow up.

Reply:
OK, we can discuss that separately, ideally together with some examples of how it could look like (although I got the idea).

Reply:
I created issue #356 to discuss this further.